setup.py

```diff
@@ -152,6 +152,12 @@ def exec_file(path_segments):
     long_description=long_description,
     long_description_content_type="text/x-rst",
     python_requires="~=3.6",
+    entry_points={
+        'console_scripts': [
+            'homeserver = synapse.app.homeserver:main',
+            'worker = synapse.app.generic_worker:main'
```

Comment:
I think we ought to be careful about name clashes.
Typing `homeserver` and `worker` on Ubuntu, I already get suggestions for packages to install.

```suggestion
            'synapse_homeserver = synapse.app.homeserver:main',
            'synapse_worker = synapse.app.generic_worker:main'
```

Otherwise we risk doing a Postgres where `createuser` is a command which actually creates a Postgres user (even though it doesn't sound like it) :).